scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -148,21 +155,40 @@ object ScaladocParser {
     }
   }
 
-  private def listBlockParser[_: P](minIndent: Int = 1): P[ListBlock] = P {
+  def listItemParser[_: P](indent: Int): P[ListItem] = {
+    def stop(indent: Int) =
+      nl ~ hspace.rep(max = indent - 1) ~ listPrefix ~ hspaces1 ~ textParser
+
+    def innerBody =
+      (!stop(indent) ~
+        listBlockParser(indent) |
+        mdCodeBlockParser(indent) |
+        tableParser(indent) |
+        textParser(indent)).rep(0, sep = paraSep)
+
+    (textParser ~ innerBody)
+      .map { case (beginning, other) => ListItem(beginning +: other) }
+  }
+
+  private def listBlockParser[_: P]: P[ListBlock] = listBlockParser(0)
+
+  private def listBlockParser[_: P](minIndent: Int): P[ListBlock] = P {
```

Comment:
should we extract listParser below as, say, listParserImpl, and define listBlockParser as `startOrNl ~ listParserImpl(0)` and name the other one as `nestedListBlockParser` and define as `nl ~ listBlockParserImpl(...)`?

Reply:
I don't think that there will be any difference. Anyway it will be used only here so I think it's ok to leave it here.